requirements/base.in

```diff
@@ -4,7 +4,6 @@ celery
 dj-database-url
 django<2.3
 django-geojson
-django-leaflet
```

Comment:
Good riddance! That's when you can `make requirements` to update all related requirement files.